client/app/scripts/superdesk-archive/controllers/upload.js

```diff
@@ -7,6 +7,8 @@ define(['lodash'], function(_) {
         $scope.items = [];
         $scope.saving = false;
         $scope.failed = false;
+        $scope.enableSave = false;
+        var requiredFields = ['headline', 'description', 'byline', 'slugline'];
```

Comment:
if `byline` is available in user profile then it should be auto-populated. This is how it behaves in case of text articles/packages. It should behave same in case of upload also.